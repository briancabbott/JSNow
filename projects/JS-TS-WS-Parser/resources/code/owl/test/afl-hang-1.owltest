---a = 'x' 'y' 'z' : a
 .operators prefix
  b : b
b = ''
