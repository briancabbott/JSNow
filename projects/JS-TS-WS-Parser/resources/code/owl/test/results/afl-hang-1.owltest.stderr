error: this grammar is ambiguous

. x y z 

  can be parsed in two different ways: as

. x y z 
  a:a-- 

  or as

.   x y z 
  b a:a-- 
  a:b---- 

